src/main/MainScreen.js

```diff
@@ -24,13 +26,18 @@ export default class MainScreen extends PureComponent<Props> {
 
   render() {
     const { styles } = this.context;
+    const { orientation } = this.props;
 
     return (
       <View style={[styles.flexed, styles.backgroundColor]}>
-        <ZulipStatusBar />
+        { orientation === 'PORTRAIT' && <ZulipStatusBar /> }
```

Comment:
You should handle this from inside the `ZulipStatusBar` component.

Reply:
@borisyankov OK, I moved it inside the component in 5adbfce. Thanks!